src/main/java/nextstep/optional/Expression.java

```diff
@@ -20,6 +22,11 @@ static Expression of(String expression) {
             }
         }
 
-        throw new IllegalArgumentException(String.format("%s는 사칙연산에 해당하지 않는 표현식입니다.", expression));
+        return Arrays.stream(Expression.values())
+                .filter((expressionValue) -> matchExpression(expressionValue, expression))
+                .findFirst()
+                .orElseThrow(() -> {
+                    throw new IllegalArgumentException(String.format("%s는 사칙연산에 해당하지 않는 표현식입니다.", expression));
+                });
```

Comment:
```suggestion
                .orElseThrow(() -> new IllegalArgumentException(String.format("%s는 사칙연산에 해당하지 않는 표현식입니다.", expression)));
```

이렇게도 해볼 수 있을 것 같아요. 😄